week-1/K-functions-parameters/exercise.js

```diff
@@ -1,6 +1,6 @@
 // Complete the function so that it takes input parameters
 function multiply() {
```

Comment:
This section is about function **parameters**, you need to use them so you can generate dynamic results.

Here's an example solution:
```
function multiply(a, b) {
  return a * b
}
```
That way you can call `function(4, 10), function(6, 7)` etc. and still return the correct value.